crates/test-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# Hieratika Test Utils
+
+Handy functions to simplify testing of Hiratika.
```

Comment:
This says nothing. It should explain what utilities it provides, primarily the E2E testing stuff.